web/src/markdown_config.ts

```diff
@@ -113,11 +63,12 @@ export const get_helpers = (): MarkdownHelpers => ({
     is_valid_user_id: people.is_known_user_id,
 
     // user groups
-    get_user_group_from_name: (name) => user_group(user_groups.get_user_group_from_name(name)),
+    get_user_group_from_name: (name: string) =>
+        user_group(user_groups.get_user_group_from_name(name)),
     is_member_of_user_group: user_groups.is_direct_member_of,
 
     // stream hashes
-    get_stream_by_name: (name) => stream(stream_data.get_sub(name)),
+    get_stream_by_name: (name: string) => stream(stream_data.get_sub(name)),
```

Comment:
Typescript asked for these, maybe we added a rule since this file was converted? I can make a separate commit where the name types are put here

Reply:
Okay nevermind apparently I don't need them, haha. I'll take them out then